File: doc/src/codes/Users_Manual/geometry_2.cpp
#include "autd3.hpp"

autd3::ControllerBuilder()
    .add_device(autd3::AUTD3(autd3::Vector3::Zero()))
    .add_device(autd3::AUTD3(autd3::Vector3(autd3::AUTD3::DEVICE_WIDTH, 0, 0))
                    .with_rotation(autd3::EulerAngles::from_zyz(
                        0 * rad, autd3::pi / 2 * rad, 0 * rad)))

File: doc/src/codes/Users_Manual/geometry_2.cs
using static AUTD3Sharp.Angle.Units;

new ControllerBuilder()
   .AddDevice(new AUTD3(Vector3d.zero))
   .AddDevice(
      new AUTD3(new Vector3d(AUTD3.DeviceWidth, 0, 0))
         .WithRotation(EulerAngles.FromZYZ(0 * Rad, AUTD3.Pi / 2 * Rad, 0 * Rad)))